debian-mk-kernel.mk

```diff
@@ -13,7 +13,7 @@ all:  bbl-q bbl-u
 	@echo
 
 $(KERNEL_IMAGE): riscv-linux/.config riscv-linux/Makefile riscv-linux
-	$(MAKE) -C riscv-linux ARCH=riscv CROSS_COMPILE=riscv64-linux-gnu- vmlinux
+	$(MAKE) -C riscv-linux ARCH=riscv vmlinux
```

Comment:
Hmm, passing `CROSS_COMPILE=riscv64-linux-gnu-` is clearly bogus, but are you 100% sure omitting `CROSS_COMPILE=...` is valid and works? Even if it does, something like

```diff
- 	$(MAKE) -C riscv-linux ARCH=riscv CROSS_COMPILE=riscv64-linux-gnu- vmlinux
+	$(MAKE) -C riscv-linux ARCH=riscv CROSS_COMPILE=$(CROSS_COMPILE) vmlinux
```
Seem safer and easier to understand to me....

Reply:
Okay, I will check that project isn't built by gcc which is installed on operating system.

Reply:
@janvrany I checked! It'okay. Because make tool set environment variable CROSS_COMPILE. CROSS_COMPILE is inherited further.

Reply:
If you would like to write `CROSS_COMPILE=$(CROSS_COMPILE)`, no problem. It is more readable :)